### DISCUSSION TITLE
Similar peptides in a same spectral library

### DISCUSSION THREAD

**Discussion by cctsou** [vdemichev/DiaNN#183] [General]
In general, how does DIA-NN handle similar peptides in the same spectral library? 

Say if we have 'AABA' and 'AAAB' peptides in a spectral library and the peptide 'AABA' is present in the data with a very good matching score, and the same peak group also has a very good matching score against 'AAAB'. Will both peptides be considered as confident ID? Or does DIA-NN also compute delta score between top peptide hits for the same peak group to resolve the ambiguous IDs?

Thanks

**Comment by vdemichev** [comment]
Sorry for the late reply. Sometimes I somehow miss posts.

If it's exactly the same peak group, only the best match will be reported. DIA-NN has an algorithm ('Remove likely interferences', enabled by default) which prevents multiple matches to the same peak group. This anlgorithm is not 100% efficient but still does a good job.

Best,
Vadim
